### PR TITLE
feat(cli): Failure messages

### PR DESCRIPTION
**Pull request by lwander**
very important

### CODE REVIEW

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java [hunk 1]
```diff
@@ -71,6 +72,20 @@ public void setColor(boolean color) {
...
 
+  private static String[] failureMessages = {
+    "I'm sorry " + System.getProperty("user.name") + ", I'm afraid I can't do that.",
+    "If at first you don't succeed, try again with a different set of flags.",
+    "Looks like you're trying to deploy Spinnaker. Would you like some help?"
+  };
+
+  private static void showRandomFailureMessage() {
+    if (ThreadLocalRandom.current().nextInt(0, 100) < 5) {
+      int index = ThreadLocalRandom.current().nextInt(0, failureMessages.length);
```
Comment (jtk54) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: Can't you do a `roll % failureMessages.length` here for the message index?

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: what's roll?

Comment (jtk54) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: `roll = ThreadLocalRandom.yadayada.nextInt(...)`

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: given that the API directly supports setting bounds on the random int this seem neaters as is

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java [hunk 2]
```diff
@@ -71,6 +72,20 @@ public void setColor(boolean color) {
 
   private static JobExecutor jobExecutor;
 
+  private static String[] failureMessages = {
+    "I'm sorry " + System.getProperty("user.name") + ", I'm afraid I can't do that.",
```
Comment (danielpeach) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: Missing: 'This mission is too important for me to allow you to jeopardize it.'

Comment (danielpeach) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: Also missing: 'I have just picked up a fault in the AE-35 unit.'

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: SGTM

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java (hunk 1 above)
Comment (jtk54) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: Something like: 
```groovy
roll = ThreadLocal.current().nextInt(0, 100)
if (roll < 5) {
    index = roll % failureMessages.length
}
```
is what I mean.

Comment (jtk54) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: Also this is fine LGTM.

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: Whenever we're inside that if statement, roll will be a random int between 5 and 0. Now index is no longer uniformly distributed, and will land on 0 and 1 twice as often as 2 given that there are 3 messages. That's why I regenerate the random number.

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java]: Of course this is the PR that we bikeshed on

**Pull request merged**